Dockerfile-dev

```diff
@@ -0,0 +1,117 @@
+# Find eligible builder and runner images on Docker Hub. We use Ubuntu/Debian instead of
+# Alpine to avoid DNS resolution issues in production.
+#
+# https://hub.docker.com/r/hexpm/elixir/tags?page=1&name=ubuntu
+# https://hub.docker.com/_/ubuntu?tab=tags
+#
+#
+# This file is based on these images:
+#
+#   - https://hub.docker.com/r/hexpm/elixir/tags - for the build image
+#   - https://hub.docker.com/_/debian?tab=tags&page=1&name=bullseye-20210902-slim - for the release image
+#   - https://pkgs.org/ - resource for finding needed packages
+#   - Ex: hexpm/elixir:1.13.2-erlang-24.2.1-debian-bullseye-20210902-slim
+#
+ARG ELIXIR_VERSION=1.13.4
+ARG OTP_VERSION=24.2.1
+ARG DEBIAN_VERSION=bullseye-20210902-slim
+
+ARG BUILDER_IMAGE="hexpm/elixir:${ELIXIR_VERSION}-erlang-${OTP_VERSION}-debian-${DEBIAN_VERSION}"
+ARG RUNNER_IMAGE="debian:${DEBIAN_VERSION}"
+
+FROM ${BUILDER_IMAGE} as dev
+
+# install build and dev dependencies
+RUN apt-get update -y && apt-get install -y \
+  build-essential curl git inotify-tools
+
+# isntall nodejs using nodesource
+# https://github.com/nodesource/distributions/blob/master/README.md#deb
+RUN curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
+RUN apt-get install -y nodejs
```

Comment:
@taylordowns2000 I would suggest changing this to the way we do it for our main Dockerfile: https://github.com/OpenFn/Lightning/blob/main/Dockerfile#L31

We don't use nodesource or deb files, we have a script that installs directly from the official binaries - this allows us to install an exact version based on an environment variable... 

It may need to be tweaked a bit to support arm64..

Reply:
we haven't had arm64 suport anyway because of the rambo dependency but it looks like 14 hours ago that outstanding PR was just merged! i'll head back to my mac in a second and test this out. very exciting.